src/extra/buttons.css

```
@@ -59,8 +59,14 @@
     --_ink-shadow: var(--_ink-shadow-dark);
   }
 
+  [data-theme=dark] {
```

nice attention to detail in the repo here, noticing buttons including dark theme related selectors 👍🏻 

```suggestion
  @nest [data-theme=dark] & {
```

that's the selector we need here, so that any ancestor with that attribute and value pair then style buttons 👍🏻 

how comes `--_bg` and `--_highlight` appear to be added?

---

I have to assume my editor was formatting on save, and there's no formatter setup that VSCode could find...